src/wooden-tree/index.js

```diff
@@ -0,0 +1,7 @@
+import { Tree } from 'react-wooden-tree';
+
+export { ActionTypes, callBack, Node, treeDataReducer } from 'react-wooden-tree';
+require('./index.scss');
```

Comment:
you can use import here too `import './index.scss'`

Reply:
Why it is better? How important it is?

Reply:
require and import are equivalent. Require is pre ES6. You should not mix them. We went with import in this repo.

Also you should call root(your index.scss) scss file `style.scss`. Once you run build it will automatically create `wooden-tree.css` file which you need to import in your app to have these styles. It will ignore anything else.

Reply:
Please do not mix require and import at the same file. Unless you really have to use `require` for instance for `requireContext`